### PR TITLE
CI: Ignore setuptools distutils warning

### PR DESCRIPTION
**Pull request by TomAugspurger**
xref https://github.com/pandas-dev/pandas/issues/35252. Just ignoring in the test. Can discuss a proper solution in the issue.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Last failure was a crashed worker (seems to be happening more often recently?) I restarted that job.

**Comment by TomAugspurger** [comment]
Travis finished, didn't report back.

Merging once the windows test finishes to get CI passing again.

File: ci/deps/azure-36-minimum_versions.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ dependencies:
 
   # tools
   - cython=0.29.16
-  - pytest=5.0.1
+  - pytest>=5.0.1, <6.0.0rc0
```
Comment (simonjayhawkins) [inline ci/deps/azure-36-minimum_versions.yaml:9]: @TomAugspurger this is minimum versions. should the pin be kept here

Comment (TomAugspurger) [reply ci/deps/azure-36-minimum_versions.yaml:9]: Ah, yeah it should be pinned exactly.

Comment (simonjayhawkins) [reply ci/deps/azure-36-minimum_versions.yaml:9]: submitted PR #35274

**Pull request merged**